### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by geeknoid**
Flatten pkg/aspect/*/* to pkg/aspect/*

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/182)

### CODE REVIEW

File: pkg/aspect/loggerManager_test.go [hunk 1]
```diff
@@ -12,10 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package logger
+package aspect
 
 import (
 	"errors"
+	"mandar/exp/pkg/aspect"
```
Comment (mandarjog) [inline pkg/aspect/loggerManager_test.go]: That can't be correct. Your IDE must have added this

**Comment by geeknoid** [comment]
PTAL.

File: pkg/adapter/testing/adapters.go [hunk 2]
```diff
@@ -12,17 +12,16 @@
...
-	"testing"
+	gt "testing"
 
 	"istio.io/mixer/pkg/adapter"
-	"istio.io/mixer/pkg/registry"
 )
 
 // RegisterFunc is the function that registers adapters into the supplied registry
-type RegisterFunc func(registry.Registrar) error
+type RegisterFunc func(adapter.Registrar) error
```
Comment (mandarjog) [inline pkg/adapter/testing/adapters.go]: Redefinition 
This function is defined in
Pkg/adapter.RegisterFunc

File: pkg/adapter/registrar.go [hunk 3]
```diff
@@ -12,29 +12,25 @@
...
-// Registrar -- Interface used by adapters to register themselves
+// Registrar is used by adapters to register themselves.
 type Registrar interface {
 	// RegisterCheckList
-	RegisterCheckList(adapter.ListCheckerAdapter) error
+	RegisterCheckList(ListCheckerAdapter) error
 
 	// RegisterDeny
-	RegisterDeny(adapter.DenyCheckerAdapter) error
+	RegisterDeny(DenyCheckerAdapter) error
```
Comment (mandarjog) [inline pkg/adapter/registrar.go]: Can we make the names uniform?
Register${aspectname} (${aspectname}Adapter)

File: pkg/aspect/BUILD [hunk 4]
```diff
@@ -18,19 +18,38 @@ ISTIO_DEPS = [
...
-    "@com_github_google_go_genproto//googleapis/rpc/code:go_default_library",
     "//pkg/expr:go_default_library",
+    "@com_github_golang_protobuf//jsonpb:go_default_library",
+    "@com_github_golang_protobuf//proto:go_default_library",
+    "@com_github_golang_protobuf//ptypes/empty:go_default_library",
+    "@com_github_golang_protobuf//ptypes/struct:go_default_library",
+    "@com_github_google_go_genproto//googleapis/rpc/code:go_default_library",
+    "@com_github_istio_api//:mixer/v1/config/descriptor",
 ] + select({
     ":local": ["@local_istio_api" + d for d in ISTIO_DEPS],
```
Comment (geeknoid) [inline pkg/aspect/BUILD]: Done

File: pkg/adapter/testing/adapters.go (hunk 2 above)
Comment (geeknoid) [inline pkg/adapter/testing/adapters.go]: Done

File: pkg/adapter/registrar.go (hunk 3 above)
Comment (geeknoid) [inline pkg/adapter/registrar.go]: Done

File: cmd/server/server.go [hunk 5]
```diff
@@ -33,7 +33,7 @@ import (
 
 	denyadapter "istio.io/mixer/adapter/denyChecker"
 	"istio.io/mixer/pkg/aspect"
-	denysupport "istio.io/mixer/pkg/aspect/denyChecker"
+	denysupport "istio.io/mixer/pkg/aspect"
```
Comment (geeknoid) [inline cmd/server/server.go]: Done.

**Comment by geeknoid** [comment]
PTAL.

**Pull request merged**